models/publication_test.go

```diff
@@ -12,12 +12,10 @@ func TestNewPublicationMinimalFields(t *testing.T) {
 	id, _ := rdf.NewIRI("http://example.com/record1")
 	title, _ := rdf.NewLiteral("Hello world!")
 	created, _ := rdf.NewLiteral("2004-06-11?")
-	identifier, _ := rdf.NewLiteral("1234567")
```

Comment:
Would you consider adding coverage of the changes you made to the pub model to this test, or do you think this is sufficiently covered already?

Reply:
So we don't have tests for any of the SetXXInfo() methods for any of the model entities. It's fairly boilerplate code, but definitely testable.

I defer to your recommendation on whether tests should be added for these methods.

Reply:
Meh, you convinced me not to care. :)